src/components/Overlay/Overlay.js

```diff
@@ -6,7 +6,7 @@ import as from "../../enhancers/as";
 import Hidden from "../Hidden";
 
 class Component extends React.Component {
-  state = {};
+  state = {wrapper: null};
```

Comment:
Thank you, @madiodio. ESLint/Prettier is complaining about the lack of spaces between those brackets.

Maybe we should add a [pre-commit hook](https://prettier.io/docs/en/precommit.html) to help people on this. @Thomazella

Reply:
I was reading about that yesterday. Should be easy.

Reply:
I don't think that option would work on GitHub or contributor's machine out of the box. It needs execute permission.

Reply:
This commit, for example, was created within GitHub.com

Reply:
🤔 Oh well.

Reply:
Let's discuss that later on Slack. XD

@madiodio, could you please fix that line?

Reply:
@diegohaz Done!